xml2rfc/writers/preptool.py

```diff
@@ -507,25 +509,25 @@ def check_ascii_text(self, e, p):
             if c.text and not isascii(c.text):
                 show = c.text.encode('ascii', errors='replace')
                 if c.tag in unicode_content_tags:
-                    if not c.get('ascii') and not c.tag in bare_unicode_tags and not is_script(c.text, 'Latin'):
+                    if c.tag in bare_unicode_tags_with_notice:
+                        if self.options.warn_bare_unicode:
+                            self.warn(c, 'Found non-ascii characters outside of elements that can have non-ascii content, in <%s>: %s' % (c.tag, show))
```

Comment:
The statement "can have" is unfounded.  But this is a warning, so maybe that is not a problem.

Reply:
How about: "Found non-ascii characters in a tag that should be inspected to ensure they are intentional"

Reply:
Sounds good.  (We have this weird definition of ASCII that includes a few beyond-ASCII characters such as NBSP, but that level of detail goes beyond the warning message.)

Reply:
Oh, and it should be "element", not "tag"...
(The content is in the element, but not in the tag, which only has the attributes.)

Reply:
I have updated the warning to:
```
Found non-ascii characters in an element that should be inspected to ensure they are intentional, in <%s>: %s
```
Example:
```
Warning: Found non-ascii characters in an element that should be inspected to ensure they are intentional, in <t>: b'The example shows how the Hebrew-language string ????'
```